_projects/README.md

```diff
@@ -0,0 +1,50 @@
+# IDEAS for OpenAstronomy
+
+Please, if you have any issue with the following steps contact your org admin or
+ask in the multi-client real-time
+[chat platform we have](http://openastronomy.org/) (Thanks @Cadair!).
+The steps are set up from the point of view of I have an idea, what do I need so
+it's online - so the hardest steps are towards the bottom, and you read only up
+to the point you feel confident.
+
+*TLDR - Are you an expert?* Fork the repo and modify
+the [template file](./_template.md) according to your idea, save it under
+`_projects/YYYY/sub-org/` with a meaningful filename and do a Pull-request.
+
+## What to I need to have an idea?
+
+You are here, so that's the first step!
+
+You need to have a description and a list of candidate mentors. Which requirements
+do you think are needed (_e.g._, knowing c++, expertise in OOP)? How difficult
+would you say it's the project? Is there already an issue related with that idea?
+
+If you have all that then think in an attractive project title and a one line
+description. Then you put all in one file as explained below.
+
+You can make your idea with very specific milestones or as open ended as you
+want. Either way is good. If it's very specific the students will know how to
+write an application as you wanted, but leave less freedom for different
+approaches.
+
+## Format needed
+
+Cool,  you've got already all the above?
+
+All you need now is to put it in a plain text file following
+the [template file](./_template.md) (you may click in raw to see how it looks).
+
+The file is composed in two parts (separated by a `---`), the top is like a yaml
+file, the bottom is in markdown format. Change the text in the template keeping
+special care of the structure of the yaml file and the headings on the markdown
+part (_e.g._, the top level starts with a 4 level title: `####`).
+
+
+## How do I do so it appears online?
+
+Are you familiar with git/github? If you are not contact us and share the above
+file with us, we will take care to put it in place.
+If, in the other hand, you are feel confident with github: go ahead, fork this
+repository, create a subdirectory for your sub-organisation (if that doesn't
```

Comment:
I think this needs to be very specific.  For example, "find the subdirectory for your sub-organisation and the year of the project (for example, if you are submitting an astropy project, go to /_projects/2017/astropy).  If your organization does not exist, create it following the form of one of the other organizations."